web/src/playground_links_popover.ts

```diff
@@ -77,6 +82,10 @@ function get_playground_links_popover_items(): JQuery | undefined {
         return undefined;
     }
 
+    if (!playground_links_popover_instance) {
+        return undefined;
+    }
```

Comment:
Similarly with this one. Are you just trying to teach the type-checker what `is_open` means? Maybe the right answer is to just inline `is_open` in these places.

Reply:
Yup, that would be the reason why I was doing as such. And inline definitely sounds better, than this.

Reply:
Updated to inline 👍🏻